stage1/init/init.go

```diff
@@ -797,6 +809,10 @@ func getContainerSubCgroup(machineID string, canMachinedRegister bool) (string,
 			return "", errwrap.Wrap(errors.New("could not get unit name"), err)
 		}
 		subcgroup = filepath.Join(slicePath, unit)
+
+		if unified {
+			subcgroup = filepath.Join(subcgroup, "payload")
```

Comment:
Note for posterity, this come from systemd-nspawn: https://github.com/systemd/systemd/blob/v231/src/nspawn/nspawn-cgroup.c#L149